src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,57 @@
+// import React, { useState } from "react";
+// import { Drawer } from "antd";
+import { useDispatch, useSelector } from "react-redux";
+import { RootState } from "../../store/store";
+import { changeToCartPage, changeToBookPage } from "../../domain/app/appSlice";
```

Comment:
I don't think so we need these actions anymore as we are using routes.

Reply:
yeah should have commented code wiz not important my bad...